src/components/Home/Introduction/index.jsx

```diff
@@ -1,38 +1,103 @@
-import { NavLink } from 'react-router-dom';
+import React, { useState } from 'react';
 import ALJNB from '../../../assets/ALJNB.jpg';
```

Comment:
Did you know that you can remove the ../../... with the last Loïc's config ?

Reply:
I didn't know! kind of sick tho! :o